AcsEmulator/AcsEmulatorAPI.Tests/AcsEmulatorAPI.Tests.csproj

```diff
@@ -10,11 +10,11 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="coverlet.collector" Version="6.0.0" />
+    <PackageReference Include="coverlet.collector" Version="6.0.4" />
```

Comment:
I haven't removed this yet but I think we can. If we want test coverage we can use something built-in